lib/middleware/request-handler.ts

```diff
@@ -87,7 +87,7 @@ export function elevateRequest (req: Interface.IOurRequest,
         });
     };
 
-    res.sendError = function(err, where, reason): Promise<void> {
+    res.sendError = function(err: any, where: string, reason: any): Promise<void> {
```

Comment:
I will vote for getting rid of `res.sendError` and let `restify` handle the sending of errors.(The method has 0 usage in our codebase now).